README.md

```diff
@@ -1,39 +1,277 @@
-> **NOTE:** It is a general template that can be used for a project README.md, example README.md, or any other README.md type in all Kyma repositories in the Kyma organization. Not all the sections are mandatory. Use only those that suit your use case but keep the proposed section order.
+# Application Connector Manager
 
-# {Project Title} (mandatory)
+## Overview
 
-> Modify the title and insert the name of your project. Use Heading 1 (H1).
+The repository contains following projects:
 
-## Overview (mandatory)
+- **Application Connector Manager** - operator compatible with Lifecycle Manager that manages Application Connector module in Kyma.
+- **Application Connector Module build configuration** to deliver the functionality of Application Connector as a Kyma module
 
-> Provide a description of the project's functionality.
->
-> If it is an example README.md, describe what the example illustrates.
+> Note: Docker images for Application Connector binaries are build separately from main GitHub [Kyma repository](https://github.com/kyma-project/kyma/)
+ 
+## How it works 
+ 
+The Application Connector Module represents a specific version of Application Connector binaries delivered via Kyma release channel.\
+The configuration of released module is described as a `ModuleTemplate` Custom Resource and delivered as OCI image.\
+It can be installed on Kyma Cluster managed by Moduletemplates operator.
```

Comment:
```suggestion
It can be installed on the Kyma Cluster managed by Template Operator.
```
Correct me if I am wrong but I think we're talking about this [Template Operator](https://github.com/kyma-project/template-operator), correct?